bench/workgen/workgen.cpp

```diff
@@ -362,19 +383,27 @@ WorkloadRunner::start_tables_create(WT_CONNECTION *conn)
             continue;
         }
 
-        for (int creates = 0; creates < _workload->options.create_count;
-             ++creates, ++total_creates) {
-            char uri[BUF_SIZE];
-            snprintf(uri, BUF_SIZE, "table:%s%" PRIu64, _workload->options.create_prefix.c_str(),
-              total_creates);
+        int creates = 0;
```

Comment:
`creates` could be a `size_t`

Reply:
```size_t``` is specifically used for the size of objects/data structures; here we are just counting the number of creates. I deliberately chose ```int``` to match ```options.create_count```. It helps to compare the same variable types so that an implicit casting is not needed.